caluma/form/tests/snapshots/snap_test_form.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# snapshottest: v1 - https://goo.gl/zC4yUc
+from __future__ import unicode_literals
+
+from snapshottest import Snapshot
+
+
+snapshots = Snapshot()
+
+snapshots["test_save_form 1"] = {
+    "saveForm": {
+        "clientMutationId": None,
```

Comment:
Could you pass a `clientMutationId` to verify if it returns the same id as passed?

Reply:
Good point. Added test.